src/coreclr/tools/Common/Compiler/Win32Resources/ResourceData.cs

```diff
@@ -67,6 +89,44 @@ public byte[] FindResource(ushort name, ushort type, ushort language)
             return FindResourceInternal(name, type, language);
         }
 
+        /// <summary>
+        /// Add or update resource
+        /// </summary>
+        public void AddResource(string name, string type, ushort language, byte[] data) => AddResourceInternal(name, type, language, data);
```

Comment:
I think these are all actually `type`, then `name` - that is the order for `AddResourceInternal`.

Reply:
First I fixed the naming, and then swapped the parameters to match with FindResource.